app/src/main/assets/json/server-config.json

```diff
@@ -620,7 +620,7 @@
       "Subscribes_to_upcoming_lives_streams": "Subscribes to upcoming live streams.",
       "Successful_projects_have_been_created_on_kickstarter": "Successful projects have been created on Kickstarter.",
       "Support_creative_independence": "Support creative independence",
-      "Support_creative_spaces_and_businesses_affected_by": "Support creative spaces and businesses affected by COVID-19. See projects near you. ▸",
```

Comment:
Did we also want to merge these changes in this PR?

Reply:
yeap, it's just updating the translations for the updated copy for lights on, and the result of executing  ```./milkrun update-strings``` . With this and the releases firebase internal you can see that all the scripts work perfectly after updating the gems :+1:

Reply:
For what it's worth - what we typically do on iOS is only commit string updates that are relevant to the changes being made in the PR. And then from time-to-time we will run an app-wide translations update and open a PR for that specifically.